bedevere/__main__.py

```diff
@@ -10,27 +10,30 @@
 from gidgethub import aiohttp as gh_aiohttp
 from gidgethub import routing
 from gidgethub import sansio
+from gidgethub import actions
 
 from . import backport, gh_issue, close_pr, filepaths, news, stage
 
-import sentry_sdk
+# import sentry_sdk
```

Comment:
Can we still use sentry in GitHub Actions? Is it needed? If not we can remove it from requirements.txt